stdlib/char.gr

```diff
@@ -1,35 +1,56 @@
+/**
+ * @module Char: Utilities for working with chars.
+ * A Grain `Char` represents a single [Unicode scalar value](https://www.unicode.org/glossary/#unicode_scalar_value)
+ * @example import Char from "char"
+ * 
+ * @since 0.3.0
+ */
+
 import WasmI32 from "runtime/unsafe/wasmi32"
 import Memory from "runtime/unsafe/memory"
 import Errors from "runtime/unsafe/errors"
 import { tagSimpleNumber, allocateChar, allocateString } from "runtime/dataStructures"
 
-export *
-
 exception MalformedUtf8
 
-// The minimum value of Unicode characters
-// Number
-let min = 0x0000
-
-// The maximum value of Unicode characters
-// Number
-let max = 0x10FFFF
-
-// Returns true if the given number is a valid Unicode scalar value.
-// @param n: Number - the value to check
-// @returns Bool
-let isValid = (n) => {
-  n >= min && (n <= 0xD7FF || n >= 0xE000) && n <= max
+/**
+ * The minimum valid Unicode charCode.
+ * 
+ * @since 0.3.0
+ */
+export let min = 0x0000
+/**
+ * The maximum valid Unicode charCode.
+ * 
+ * @since 0.3.0
+ */
+export let max = 0x10FFFF
+
+/**
+ * Determines whether the given character code is a valid Unicode Scalar value.
+ * 
+ * @param charCode: The value to check
+ * @returns `true` if the number refers to a valid Unicode character, `false` otherwise
+ * 
+ * @since 0.3.0
+ */
+export let isValid = (charCode) => {
+  charCode >= min && (charCode <= 0xD7FF || charCode >= 0xE000) && charCode <= max
 }
 
-// Returns the Unicode code point for the character
-// @param char: Char - the input character
-// @returns Number
+/**
+ * Determines the Unicode code point for the given character.
```

Comment:
```suggestion
 * Determines the Unicode code point for the given Char.
```
Unsure if we should use "character" or "Char"

Reply:
I prefer char because we use it more often.